### ISSUE TITLE
Control over the "call" and "write" phrase

### ISSUE THREAD

**Issue by markuman** [SergeyMosin/Appointments#523]
https://github.com/SergeyMosin/Appointments/blob/370db1ca0ac992bdd99a07572e497f821d2a4f34/translationfiles/templates/appointments.pot#L225-L236

I'd like to know what values/variables are used? Is it when the Users profiles has filled out a phone number?

**Comment by markuman** [comment]
I've create two appointments, one with phonenumber

```sql
select json_extract(data, '$.phone') from oc_appointments_pref_v2 oapv where token = 'rGbtrds4t1VCV88' ;
```

```
json_extract(data, '$.phone')|
-----------------------------+
"03012345678"                 |
```

and one without

```
json_extract(data, '$.phone')|
-----------------------------+
                             |
```

The first one sends reminder information with the call phrase at its end.  

> Wenn du den Termin verschieben musst, rufe bitte 03012345678 an.

The second one also. I think it is a bug and the second one is choosing the wrong template.

> Wenn du den Termin verschieben musst, rufe bitte an.

So it does not make differences here  
https://github.com/SergeyMosin/Appointments/blob/370db1ca0ac992bdd99a07572e497f821d2a4f34/l10n/de_DE.js#L19C1-L19C8

like it did here  
https://github.com/SergeyMosin/Appointments/blob/370db1ca0ac992bdd99a07572e497f821d2a4f34/lib/Backend/DavListener.php#L1023-L1033

**Comment by SergeyMosin** [comment]
The `$org_phone` refers to the Organizer phone. If you add a phone number in `Settings > Contact Information > Phone` it will be shown to the attendees, but if the organizer does not provide a phone number then there is nothing to show.

**Comment by markuman** [comment]
Yes, it `$org_phone` is defined as `03012345678`, it will print 

> ... If you need to reschedule, please call 03012345678.

But if it is not defined, it will print.

> ... If you need to reschedule, please call.

I would expect that it will print 

> ... If you need to reschedule, please reply to this Email.

**Comment by SergeyMosin** [comment]
I see now, this is for reminder email. The bug is actually here:

https://github.com/SergeyMosin/Appointments/blob/696c0adeaaed978731483b32f32f30baf8b26074/lib/Backend/DavListener.php#L323-L326

The fix will be available in the next release.

Thank you for the report.

**Comment by SergeyMosin** [comment]
Fixed in v2.1.5
